source/wp-content/themes/wporg-parent-2021/sass/blocks/_spinner.scss

```
@@ -0,0 +1,49 @@
+.wporg-spinner {
+	--local-size: 16px;
+	--spinner-color: var(--wp--preset--color--blueberry-1, currentColor);
+	--background-stroke-color: rgb(221, 221, 221);
+
+	width: var(--local-size);
+	height: var(--local-size);
+	display: inline-block;
+	background-color: transparent;
+	position: relative;
+}
+
+/* Static outer circle */
+.wporg-spinner::before {
+	content: '';
```

stylelint - this should use double-quotes
```suggestion
	content: "";
```